Modix/ModixBot.cs

```diff
@@ -184,7 +184,11 @@ public ModixBot(ModixConfig config, ILogger logger)
             _map.AddSingleton<InviteLinkHandler>();
             _map.AddScoped<IBehaviourConfigurationRepository, BehaviourConfigurationRepository>();
             _map.AddScoped<IBehaviourConfigurationService, BehaviourConfigurationService>();
-            _map.AddSingleton<IBehaviourConfiguration, Services.BehaviourConfiguration.BehaviourConfiguration>();
+            _map.AddSingleton<IBehaviourConfiguration, BehaviourConfiguration>();
+
+            _map.AddSingleton<INoteCreatorService, NoteCreatorService>();
+            _map.AddSingleton<INoteRetrieverService, NoteRetrieverService>();
+            _map.AddSingleton<INoteRemoverService, NoteRemoverService>();
```

Comment:
Can these be scoped instead of singletons? ModixContext is transient, so each of these services will keep their own instances of the context alive for the duration of the app.